### DISCUSSION TITLE
Amiga version

### DISCUSSION THREAD

**Discussion by FreeMEM** [vii1/DIV#50] [General]
Se que es muy pronto para este port, pero... vamos a estar atentos un gran grupo de gente cuando esto ocurra. Amiga Rulez :-)

**Comment by AFloresc** [comment]
Pero que muy atentos vamos a estar!!!

**Comment by vii1** [comment]
Sería un sueño hecho realidad. Como dices, queda un largo camino antes de que ocurra, pero me alegra mucho ver que hay más gente interesada :D

**Comment by Zardoz89** [comment]
Solo diré que tengo un A1200 a medio metro si hay que hacer pruebas.

Para poder hacer esto, antes habría que meter toda una capa de abstracción entre DIV y el hardware o una librería que abstraiga del hardware (Allegro ?)
